aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/js/dependsOnActionRegistry.js

```diff
@@ -38,12 +39,41 @@
             return action;
         }
 
+        /**
+         * Function to remove symbols that don't match the pattern
+         * @param {string} target - target string
+         * @param {RegExp} pattern -  pattern for correct symbols
+         * */
+        static _replaceForbiddenSymbols(target, pattern) {
```

Comment:
There is no need to mark pure function via private notation.
Private notation - means field or function should not be called outside as it can broke something it will be used not in the right "context". Pure function can not do anything bad if it used outside